src/coreComponents/physicsSolvers/wavePropagation/AcousticWaveEquationSEMKernel.hpp

```diff
@@ -20,6 +20,7 @@
 #define GEOSX_PHYSICSSOLVERS_WAVEPROPAGATION_ACOUSTICWAVEEQUATIONSEMKERNEL_HPP_
 
 #include "finiteElement/kernelInterface/KernelBase.hpp"
+#include "WaveSolverBase.hpp"
```

Comment:
Do you remember why you needed to include `WaveSolverBase.hpp` in the kernel file? Usually, we try not to include the solver files in their kernel files, otherwise we end up with circular dependencies.

Reply:
This is probably a left-over from an early version were I needed to include `WaveSolverBase.hpp` in the kernel file. I have removed it.